ibm/resource_ibm_cis_rate_limit.go

```diff
@@ -338,220 +338,229 @@ func resourceIBMCISRateLimitValidator() *ResourceValidator {
 	return &ibmCISRateLimitResourceValidator
 }
 func resourceIBMCISRateLimitCreate(d *schema.ResourceData, meta interface{}) error {
-	cisClient, err := meta.(ClientSession).CisAPI()
+	cisClient, err := meta.(ClientSession).CisRLClientSession()
 	if err != nil {
 		return err
 	}
+
 	cisID := d.Get("cis_id").(string)
-	zoneID, _, err := convertTftoCisTwoVar(d.Get("domain_id").(string))
-	if err != nil {
-		return err
-	}
+	zoneID := d.Get("domain_id").(string)
```

Comment:
revert back the change..
get zone_id as a combination

Reply:
done